### PR TITLE
fix: remove padding-left from last navbar item

### PR DESCRIPTION
**Pull request by lex111**
| Before   | After    |
| -------- | -------- |
| image | image |

@yangshun I'm actually wondering why `padding` is used for navrbar item links rather than regular `margin-left`? It probably isn't very useful to increase the clickable area of a navbar link in this way. Maybe we should replace `padding` with `margin`? (Then, we don't need to use new `last-child` rule.)

### CODE REVIEW

**Review by yangshun — CHANGES_REQUESTED** [review]
It's intentional, all the nav items have some padding around it to increase the clickable area,

**Comment by lex111** [comment]
Alright, but for the last navbar item can we remove `padding-left` as proposed in this PR?

File: packages/core/styles/components/navbar.pcss [hunk 1]
```diff
@@ -122,6 +122,10 @@
       flex: 0 0 auto;
       justify-content: flex-end;
     }
+
+    > :last-child {
```
Comment (yangshun) [inline packages/core/styles/components/navbar.pcss:126]: Maybe this should only be for items in the `--right` side

**Review by yangshun — APPROVED** [review]
Left a comment.

File: packages/core/styles/components/navbar.pcss (hunk 1 above)
Comment (lex111) [reply packages/core/styles/components/navbar.pcss:126]: Agree, will do it. BTW what do you think about this PR #121?

**Pull request merged**